### PR TITLE
Update to fix visual mismatch reported by Andre P

### PR DESCRIPTION
**Pull request by gcastill0**
Fixing a rogue image display, and a tab missaignment in all three tracks. Thanks to Andre P for reporting.

### CODE REVIEW

File: instruqt-tracks/terraform-cloud-azure-v2/13-private-module-registry/assignment.md [hunk 1]
```diff
@@ -104,13 +104,11 @@ timelimit: 1800
     font-size: 12px;
     font-weight: bold;
     align-items: center;
-    justify-content: center;
     height: 24px;
   }
 
-  w > img {
+  w > a img {
```
Comment (omkensey) [inline instruqt-tracks/terraform-cloud-azure-v2/13-private-module-registry/assignment.md:110]: Do the other tracks' CSS blocks need to change from `w > img` to `w > a img` as well?

Comment (gcastill0) [reply instruqt-tracks/terraform-cloud-azure-v2/13-private-module-registry/assignment.md:110]: All of them are updated.

Comment (gcastill0) [reply instruqt-tracks/terraform-cloud-azure-v2/13-private-module-registry/assignment.md:110]: I added the missing tag, @omkensey. Thanks for flagging it.

File: instruqt-tracks/terraform-cloud-aws-v2/01-setup-our-environment/assignment.md [hunk 2]
```diff
@@ -60,12 +60,12 @@ timelimit: 1800
     padding: 2px 10px 2px 5px;
     font-size: 14px;
     letter-spacing: 1.2px;
-    align-items: center;
```
Comment (omkensey) [inline instruqt-tracks/terraform-cloud-aws-v2/01-setup-our-environment/assignment.md:63]: The presence of this line in this block is different between different versions of the track.  In AWS it's been removed but in Azure and GCP it's still there.  All tracks have `justify-content: center;` in this block.

File: instruqt-tracks/terraform-cloud-aws-v2/13-private-module-registry/assignment.md [hunk 3]
```diff
@@ -105,13 +105,11 @@ timelimit: 1800
     font-size: 12px;
     font-weight: bold;
     align-items: center;
-    justify-content: center;
```
Comment (omkensey) [inline instruqt-tracks/terraform-cloud-aws-v2/13-private-module-registry/assignment.md:108]: The presence of these two lines in this block is different between different versions of the track.  In AWS and Azure, the `align-items` line is present; in GCP the `justify-content` line is present.

**Review by omkensey — COMMENTED** [review]
Couple of consistency notes on the alignment and justification definitions.

File: instruqt-tracks/terraform-cloud-aws-v2/01-setup-our-environment/assignment.md (hunk 2 above)
Comment (gcastill0) [reply instruqt-tracks/terraform-cloud-aws-v2/01-setup-our-environment/assignment.md:63]: Thanks again, @omkensey . Added the right property for flex alignment.

**Pull request closed without merging**